packages/types/src/definitions/core/OntimeEvent.type.ts

```diff
@@ -39,6 +39,7 @@ export type OntimeEvent = OntimeBaseEvent & {
   colour: string;
   revision: number;
   delay?: number; // calculated at runtime
+  dayOffset?: number; // calculated at runtime
```

Comment:
suggestion: I wonder if we are starting to complicate our code base a bit with these types

would it be better to have a `OntimePersistedEvent` and an `OntimeEvent`, where the `OntimeEvent`contains all the stuff calculated in `rundownCache`?

Reply:
Yeah tat sounds like a good idea
I've been struggling a bit with the optional so making a new type would be great 

Should we make a new type for pre or post cache.generate ?

Reply:
I am not sure if I understood. but I think the two types we need are
- one type with the base data we need, this is stored in the json file
- one type with added metadata, this will be in all the runtime stores and generated in the rundownCache

Reply:
should the "normal" `OntimeEvent` type be the on with meta data since that is what is being operated on in most of the code base, and then we create a new `dbOntimeEvent` with out the metadata?

Reply:
I am not sure if there is something definitely better. I think your suggestion is good and makes sense